### ISSUE TITLE
Problem doing a post request under 'api.auth' middleware

### ISSUE THREAD

**Issue by intrepido** [jadjoubran/laravel5-angular-material-starter#185]
I am doing the Tutorial for the version 3.0. But when I do the post request

```
$api->group(['middleware' => 'api.auth'], function ($api) {
    $api->post('posts', 'CreatePostController@create');
});
```

```
      API.all('posts').post(data).then(function(response){
           ToastService.show('Post added successfully');
       }); 
```

I get this error 

> {
>   "message": "Failed to authenticate because of bad credentials or an invalid authorization header.",
>   "status_code": 401
> }

I think this 'api.auth' middleware verify if the request has the JWT token, but in what moment this was created? How I can create the token in the application? For manage token is not the best option to use angular 'Satellizer' for store the token in Local Storage?

**Comment by jadjoubran** [comment]
Hi @intrepido 
Thank you for reporting this issue. Indeed there shouldn't be the `api.auth` in the tutorial.
And yes this will be possible in #151 and/or #41

**Comment by jadjoubran** [comment]
Fixed in 3.0 and 3.1 docs :+1: 
Feel free to open a new issue :smile:
